backend/common/src/main/java/ai/verta/modeldb/common/CommonUtils.java

```diff
@@ -65,6 +66,18 @@ public static String getStringFromProtoObject(MessageOrBuilder object) {
     }
   }
 
+  public static void logAmazonServiceExceptionErrorCodes(Logger LOGGER, AmazonServiceException e) {
+    LOGGER.info("Amazon Service Status Code: " + e.getStatusCode());
+    LOGGER.info("Amazon Service Error Code: " + e.getErrorCode());
+    LOGGER.info("Amazon Service Error Type: " + e.getErrorType());
+    LOGGER.info("Amazon Service Error Message: " + e.getErrorMessage());
```

Comment:
Is info the right level for this? If this is truly for errors, should it at least be at WARN? Or if it's not actually errors, maybe DEBUG would be better.

Reply:
Added logs with DEBUG